geode-common/src/main/java/org/apache/geode/cache/ExpirationAction.java

```diff
@@ -164,7 +164,7 @@ public static ExpirationAction fromXmlString(String xmlValue) {
 
   // The 4 declarations below are necessary for serialization
   private static int nextOrdinal = 0;
-  public final int ordinal = nextOrdinal++;
+  public int ordinal = nextOrdinal++;
```

Comment:
Why remove final from `public int ordinal` if there's no code that reassigns it?

If this is the fix for a serialization bug, then I suggest filing a separate ticket and PR for that bug with a test for it.

Reply:
Good catch! I must have accidentally removed final from this ordinal while I was fixing compilation errors related to adding other final clauses (I tried to make nextOrdinal final). I double checked the diff, this is the only place I removed final from something that was. I'll fix it.